### ISSUE TITLE
Check boxes are nearly invisible

### ISSUE THREAD

**Issue by chuckwagoncomputing** [obelisk79/OpenTheme#76]
These boxes are both checked, but you can barely tell.
![Screenshot_20240612_185650](https://github.com/obelisk79/OpenTheme/assets/1531127/241a8c84-ddbe-4395-ba72-0c3f31908ccf)

**Comment by obelisk79** [comment]
I am unable to reproduce this issue. I did recently make some corrections regarding checkbox styles, so if you could update and retest it might help. Also, share your system information might offer some insight.

**Comment by leoheck** [comment]
@chuckwagoncomputing are you using AppImage too? Waht is your Freecad version?

**Comment by leoheck** [comment]
I also noticed that the check box of the second item above is under the word `start`.

**Comment by leoheck** [comment]
The issue here looks fixed on my version
![image](https://github.com/obelisk79/OpenTheme/assets/1277920/3d3d4242-bc78-4c46-955f-e678e0ffad72)

```
OS: Ubuntu 24.04 LTS (ubuntu:GNOME/ubuntu)
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.38001 (Git) AppImage
Build type: Release
Branch: main
Hash: 1fc6da959b936a2999774a4fe5b4c4b12bf6ac8d
Python 3.11.9, Qt 5.15.13, Coin 4.0.2, Vtk 9.2.6, OCC 7.7.2
Locale: English/United States (en_US)
Installed mods: 
  * POV-Ray-Rendering
  * Assembly4 0.50.12
  * InventorLoader 1.4.0
  * ose-piping
  * OpenTheme 2024.6.22
  * SheetMetal 0.4.20
  * fasteners 0.5.24
  * asm4_disable_version_check_popup
  * Render 2024.6.5
  * OpticsWorkbench 1.0.18
  * lattice2 1.0.0
  * kicadStepUpMod
```

**Comment by obelisk79** [comment]
Indeed, I did spend some time to resolve these issues, I think there may have been some duplicate issues so I missed closing this one. Thank you for pointing it out.

**Comment by chuckwagoncomputing** [comment]
> I am unable to reproduce this issue. I did recently make some corrections regarding checkbox styles, so if you could update and retest it might help. Also, share your system information might offer some insight.

Sorry I didn't respond to this. I forgot about it until Leo's comments. I'm running 0.21.2 from the Archlinux repo. When I get a chance (probably tomorrow) I'll try different system themes to see if that's affecting it.
I'm at work now, and it seems the same issue exists on Windows, on FreeCAD 0.21.2 downloaded from freecad.org. I don't see any difference between dark or light system themes.
![Screenshot 2024-07-03 080141](https://github.com/obelisk79/OpenTheme/assets/1531127/8305484f-550f-4dd4-8aa1-9e32d99ccde2)

**Comment by chuckwagoncomputing** [comment]
I tried to debug this earlier, but I was confused by this line, which seems to reference a SVG that doesn't exist:
https://github.com/obelisk79/OpenTheme/blob/f624b1f3f8b99dc4486ce5d49cac9240041b1966/OpenDark/OpenDark.qss#L458
Is there some magic that I don't know about, or is this a mistake?

**Comment by obelisk79** [comment]
> I tried to debug this earlier, but I was confused by this line, which seems to reference a SVG that doesn't exist:

This was indeed an issue, but I corrected it about two weeks ago, make sure you've updated the themes to the most recent version using the addon-manager.

**Comment by chuckwagoncomputing** [comment]
OpenThemes wasn't installed on that computer until this morning, so it's the most recent version that Addon Manager knows about; the date/version according to Addon Manager  is 6/22, which is also the date of the most recent commit.

**Comment by obelisk79** [comment]
Interesting that the file was still missing. I had identified this and thought I had resolved it already. I reuploaded the missing images and updated the theme version to today. It *should* be ok now. Just let me know.

**Comment by chuckwagoncomputing** [comment]
It's not fixed on my Linux computer at home; not sure about Windows.
I found that if I export the svg to png and change the extension in OpenDark.qss, it works. I also tried altering the svg to see if it was being re-colored somehow, but that didn't change anything. So, it seems that the svg is not being used at all. I do have qt-svg installed.

Changing the system theme seems to have no effect on the checkboxes.

**Comment by chuckwagoncomputing** [comment]
It turns out that renaming the svg something else (and changing it in OpenDark.qss) fixes it. It seems to be a hard coded thing somewhere, but I have no idea where. I tried renaming light_up_arrow.svg check_light.svg and change all references to light_up_arrow.svg in OpenDark.qss to check_light.svg, and all the up arrows were replaced with the black check mark.
![Screenshot_20240704_151837](https://github.com/obelisk79/OpenTheme/assets/1531127/5b7099dd-0a26-4e78-b337-cf8bcb1c8413)
